docs/learn/about-iota/why-is-iota-feeless.md

```diff
@@ -5,18 +5,45 @@ description: The unique feature of IOTA is the absence of any Fee's in the proto
+
+Block Body: Protocol-dependent number of bytes, often 1,000,000 or more, of transaction data.
+
+We mentioned mining several times above. Here we look at what that means for cryptocurrencies built on a proof of work model. When a given block has just been added to the blockchain, a race begins. In this race, to mine the next block, the participants -- most commonly referred to as miners -- gather as many open transactions as they can into a new block body and calculate the Merkle Root from that, and they calculate other information for their new block header as well, all except the 4-byte nonce we mentioned above.
+
+The miner can set this nonce to any value they like, but the goal in doing so is to make the hash of the block header meet the target difficulty. To slightly oversimplify it, the nonce is changed and the header hashed repeatedly until the resulting hash begins with enough zeros. At current difficulty, approximately 1 in every 75,000,000,000,000,000,000,000 possible headers will produce a hash with enough leading zeros to be valid. When one is found, the block is mined and can be sent to the nodes where it is added to the blockchain, and the race begins again for the next block.
+
+This race to mine the next block is a key security feature in blockchain. If one could mine blocks at will, an attacker could make a competing blockchain and, by getting enough nodes to publish it, could create fraudulent transactions, reverse legitimate ones, or any of a number of other things that would rapidly cause trust in the system and the value of the associated coin to trend to 0.
+
+Because each block in the blockchain contains a double hash of the header of the previous block, any change made earlier in the blockchain requires all subsequent blocks be changed also, and these changes require the newly-changed blocks be mined again. It is easy to see that this system provides security not only for new transactions, but that the older a transaction in the blockchain is, the more secure it becomes.
+
+So miners are critical to blockchain, but mining is expensive work and must be financially compensated.
+
+When a block is mined, the successful miner is allowed to add a number of coins -- the block reward -- to his or her own wallet, the number being set by the protocol. These newly minted coins can be sold by the miner and are currently the primary way the miners are paid for their work. Unfortunately for miners, many crypto protocols reduce the size of the block reward over time. But there is another way miners can still get paid.  This is where the user comes in.
```

Comment:
When a block is mined, the successful miner is allowed to add a number of coins -- the block reward -- to his or her own wallet, the number is set by the protocol. These newly minted coins can be sold by the miner and are currently the primary way the miners are paid for their work. Unfortunately for miners, many crypto protocols reduce the size of the block reward over time. But there is another way miners can still get paid.  This is where the user comes in.